site/src/components/AvatarCard/AvatarCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { type ReactNode } from "react";
+import { Avatar } from "components/Avatar/Avatar";
+import { type CSSObject, useTheme } from "@emotion/react";
+import { colors } from "theme/colors";
+
+type AvatarCardProps = {
+  header: string;
+  imgUrl: string;
+  altText: string;
+
+  subtitle?: ReactNode;
+  maxWidth?: number | "none";
+};
+
+export function AvatarCard({
+  header,
+  imgUrl,
+  altText,
+  subtitle,
+  maxWidth = "none",
+}: AvatarCardProps) {
+  const theme = useTheme();
+
+  return (
+    <article
```

Comment:
Why would this be an article? I thought it was only used for "large" text content or "previews".

Reply:
So it's more of a hokey example, because there isn't that much content in it (for now). But the way I think about the `<article>` element is like an "article of clothing" – it's a self-contained unit of information, no matter how long it actually is